pep-0567.rst

```diff
@@ -203,9 +224,11 @@ be contained in the ``ctx`` context::
 
     def function():
         assert var.get() == 'spam'
+        assert ctx[var] == 'spam'
```

Comment:
I'd much prefer it if instead of `assert` statements you'd use comments e.g. `# Now ctx[var] == 'spam'`. (The first `assert` in the PEP is a pre-condition, you can phrase that as `# Assume <blah>`.)

Reply:
Fixed all asserts.